src/packages/cli/src/__tests__/studio.test.ts

```diff
@@ -21,8 +15,8 @@ const setupWS = (): Promise<WebSocket> => {
         /* eslint-disable-next-line @typescript-eslint/no-unsafe-assignment */
         const message: any = JSON.parse(data)
 
-        expect(message).toHaveProperty('channel')
-        expect(message).toHaveProperty('action')
+        assert.notStrictEqual(message.channel, undefined)
+        assert.notStrictEqual(message.action, undefined)
```

Comment:
Why using `assert.notStrictEqual` through the tests? Could it be something else than undefined?

Reply:
Oh it actually doesn't matter what the value is, as long as the property exists, hence the assertion.

Reply:
Ok so it checks that it's not undefined then.

It's quite weird to me to read 🤔 

Could we replace that by `assert.ok(message.action)` checking if value is truthy?

Reply:
Yep, can do! I think I did this initially because I use the same kind of assertions on other places in the test too.  (Also, I agree about the readability, but no choice :P)

Reply:
I've changed all `notStrictEqual` assertions to `assert.ok`s. Explicitly checking for `undefined` because I do not trust functions that operate on "truthy" values in JS